src/js/main.js

```diff
@@ -0,0 +1,167 @@
+/*IMPORTS*/
+import loginUser from "./login.js"
+import {createComment, editComment, deleteComment} from './api-communication.js'
+import {getPosts, createPost, editPost, deletePost} from './api-communication.js'
+import {createPostCard} from './post.js'
+
+/*VARIABLES*/
+const postsCont = document.getElementById('main__posts')
+const loginBtn = document.getElementById('loginButton')
+const saveComment = document.getElementById('editModal__save')
+const deleteCommentBtn = document.getElementById('deleteModalBtn');
+const createPostButton = document.getElementById('createPostButton');
+
+window.onload = async function (){
+    if (!sessionStorage.getItem('userId')){
+        defaultUser()
+    }
+    await displayPosts()
+    getUserLoginImg()
+}
+
+//Default user
+const defaultUser = () => {
+    sessionStorage.setItem('username', "pikachu")
+    sessionStorage.setItem('userId', 25)
+    sessionStorage.setItem('avatarimg', "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/25.png")
+}
+
+//Load userInfo from sessionStorage
+const getUserLoginImg = () =>{
+    const userDiv = document.getElementById('dropdownMenuLink')
+    const createPostAvatar = document.getElementById('createPostAvatar')
+    userDiv.textContent = sessionStorage.getItem('username')
+    createPostAvatar.src = sessionStorage.getItem('avatarimg')
+}
+
+//Create Post
+createPostButton.addEventListener('click', async () => {
+    const textPost = document.getElementById('create-post__text').value;
+    const response = await createPost(textPost)
+    location.reload()
+})
+
+//Login
+loginBtn.addEventListener('click', () => {
+    const userName = document.getElementById('login__Username');
+    loginUser(userName.value)
+})
+
+//Show/hide post comments
+postsCont.addEventListener('click', (e) => {
+
+    if (e.target.hasAttribute('data-hide-comments')){
+        const postId = e.target.getAttribute('data-show-comments')
```

Comment:
Line 54 and 63 are repeated, maybe you can take it out the if else statement and put it at the beginning of the function.